pkg/image/association.go

```diff
@@ -300,7 +304,34 @@ func AssociateImageLayers(rootDir string, imgMappings map[string]string, images
 			idx = idIdx
 		}
 		tagOrID := dirRef[idx+1:]
+
+		// afflom - Origin mappings cutoff the arch part of the filename.
+		// this adds it back in as needed.
```

Comment:
@afflom rather than including okd specifc login in `associations.go` I think this should be done when building the mapping passed to `AssociateImageLayer`. WDYT?

Reply:
@jpower432 , I agree. I never wanted this in association.go, but I don't understand how to implement this in bundle.getMapping()